crates/test-utils/src/deploy_helpers.rs

```diff
@@ -803,14 +721,11 @@ impl KakarotTestEnvironmentContext {
             evm_contracts.insert(contract_name.to_string(), contract);
         }
 
-        let starknet_provider = Arc::new(JsonRpcClient::new(HttpTransport::new(sequencer.url())));
-
-        let chain_id = starknet_provider.chain_id().await.unwrap();
-
         let deployer_account: DeployerAccount =
             serde_json::from_value(contracts.get("DeployerAccount").unwrap().to_owned())
                 .expect("Failed to fetch Deployer Account");
         let local_wallet = LocalWallet::from_signing_key(SigningKey::from_secret_scalar(deployer_account.private_key));
+        let chain_id = starknet_provider.chain_id().await.unwrap();
```

Comment:
we use the starknet chain_id ?

Reply:
yeah we use it to create the deployer_account (starnet account) on line 730